Shoko.Server/nlog.config

```diff
@@ -7,6 +8,13 @@
         <target name="console" xsi:type="ColoredConsole" layout="${date:format=HH\:mm\:ss}| --- ${message}" />
     </targets>
     <rules>
+        <!--Skip non-critical Microsoft logs and so log only own logs (Blackhole) -->
+        <logger name="Microsoft.*" maxlevel="Info" final="true" />
+        <logger name="System.Net.Http.*" maxlevel="Info" final="true" />
```

Comment:
Wouldn't this exclude the requests/responses from the log? (They're very useful for debugging)

Reply:
The "`System.Net.Http.*`" rule i mean

Reply:
Yes they are excluded, but they are already excluded in current build. The log level was set to error and above in the code, so none of the info logging got through. There shouldn't be any change and now you can enable that logging by changing the config line without rebuilding.

Reply:
I've re-enabled the http log spam in debug build as the objective of this pull request is not to change logging behavior. Also, turns out I didn't see http logs coming from System.Net.Http, it was from Microsoft.AspNetCore.